controllers/updateservice_controller.go

```diff
@@ -125,11 +125,16 @@ func (r *UpdateServiceReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, err
 	}
 
+	clusterCM, err := r.findTrustedClusterCAConfig(ctx, reqLogger, instanceCopy)
```

Comment:
Should we handle non-IsNotFound errors here?

Reply:
not sure what to do with other errors. 
This basically checks if the config map for the ca cert exists, and if not, it creates one.

Reply:
Seems like the error conditions above are handled by propagating the error: `return ctrl.Result{}, err`? So we could cope with `IsNotFound` and propagate remaining errors?